sklearn/conftest.py

```diff
@@ -28,6 +29,29 @@
         "at least pytest >= {} installed.".format(PYTEST_MIN_VERSION)
     )
 
+scipy_datasets_require_network = sp_version >= parse_version("1.10")
+
+
+def raccoon_face_or_skip():
+    if scipy_datasets_require_network:
+        run_network_tests = environ.get("SKLEARN_SKIP_NETWORK_TESTS", "1") == "0"
+        from scipy.datasets import face
+
+        try:
+            return face(gray=True)
+        except ImportError as e:
+            if run_network_tests:
+                raise ValueError(
+                    "pooch is required when SKLEARN_SKIP_NETWORK_TESTS=0"
+                ) from e
```

Comment:
The previous behavior was to skip instead of error when pooch is not installed. We usually skip on soft dependencies not being installed